src/lib/features/auth/specs/auth.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { get } from 'svelte/store';
```

Comment:
hate to tell you this, but this won't be necessary :p. I implemented it in #126 (which you can review, btw)